### ISSUE TITLE
How to use monai.losses.TverskyLoss

### ISSUE THREAD

**Issue by Guoxt** [Project-MONAI/MONAI#6679]
A 2D segmentation model with a network output of 2-channel logical values without softmax processing. For example, the network output dimension is (B, 2128128). And the dimension of the target is (B, 1128128). How to set the parameters of the Loss function in this case?
